GitUI/Editor/FileViewer.cs

```diff
@@ -600,7 +600,9 @@ private string GetFileText(string fileName)
 
             if (File.Exists(path))
             {
-                using (var reader = new StreamReader(path, Module.FilesEncoding))
+                // StreamReader disposes of 'fileStream'.
+                var fileStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Comment:
Yes I'm on it. How do I do that? I just change it in my fork, push it to my fork and it's done?